tests/shared/src/test/scala/scala/meta/tests/parsers/ScaladocParserSuite.scala

```diff
@@ -206,6 +206,44 @@ class ScaladocParserSuite extends FunSuite {
     assertEquals(result, expectation)
   }
 
+  test("markdown code blocks") {
+    val codeBlock = // keep all newlines and leading spaces
+      """|  object Foo {
+         |    def bar(): String = "foobar"
+         |  }
+         |""".stripMargin.split("\n")
+    val codeBlockAsComment = codeBlock.mkString("\n *")
+
+    val result =
+      parseString(
+        s"""
+          /**
+            *  ```scala sc:compile
```

Comment:
both scala and sc:compile will be lost, is that ok?

Reply:
I think it's a good idea to create a new `MarkdownCodeBlock` element (as it was already suggested) that will store information about language and options because of course, we need them.